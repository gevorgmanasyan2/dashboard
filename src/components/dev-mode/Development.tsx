import { WarningFilled } from '@ant-design/icons'
import React, { FC } from 'react'
import './development.scss'

const Development: FC = () => {
  return (
    <div className="development__container">
      <WarningFilled />
      <span>{process.env.NODE_ENV}</span>
      <WarningFilled />
    </div>
  )
}

export default Development
