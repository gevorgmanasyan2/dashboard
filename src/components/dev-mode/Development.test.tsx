/* eslint-disable no-undef */
import React from 'react'
import { shallow } from 'enzyme'
import Development from './Development'

it('should render Development component', () => {
  expect(shallow(<Development />)).toMatchSnapshot()
})
