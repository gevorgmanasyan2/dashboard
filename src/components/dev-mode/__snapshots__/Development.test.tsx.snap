// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Development component 1`] = `
<div
  className="development__container"
>
  <ForwardRef(WarningFilled) />
  <span>
    test
  </span>
  <ForwardRef(WarningFilled) />
</div>
`;
